.github/dependabot.yaml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
Not a blocking issue but I have noticed that several repositories in the CoP have begun to integrate [renovate](https://github.com/renovatebot/renovate)